src/config.rs

```diff
@@ -1,11 +1,16 @@
-use diesel::pg::PgConnection;
-use diesel::prelude::*;
-use dotenv::dotenv;
-use std::env;
+//use diesel::pg::PgConnection;
+//use diesel::prelude::*;
+//use dotenv::dotenv;
+//use std::env;
+//use r2d2_postgres::PostgresConnectionManager;
+//
+//pub fn establish_connection() -> PgConnection {
+//    dotenv().ok();
+//
+//    //let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
+//
+//    let database_url = "127.0.0.1:5432";
+//    PgConnection::establish(database_url).expect(&format!("Error connecting to {}", database_url))
+//}
 
-pub fn establish_connection() -> PgConnection {
-    dotenv().ok();
-
-    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
-    PgConnection::establish(&database_url).expect(&format!("Error connecting to {}", database_url))
-}
+pub const SECRET: &str = "secret";
```

Comment:
Make this an environment variable, and unwrap it without a default specified for added security